src/GitHub.InlineReviews/InlineReviewsPackage.cs

```diff
@@ -1,24 +1,33 @@
 ﻿using System;
+using System.ComponentModel.Design;
 using System.Runtime.InteropServices;
+using System.Threading;
 using GitHub.InlineReviews.Commands;
 using GitHub.InlineReviews.Views;
+using GitHub.Services;
 using GitHub.VisualStudio;
+using Microsoft.VisualStudio.ComponentModelHost;
 using Microsoft.VisualStudio.Shell;
 using Microsoft.VisualStudio.Shell.Interop;
+using Task = System.Threading.Tasks.Task;
 
 namespace GitHub.InlineReviews
 {
-    [PackageRegistration(UseManagedResourcesOnly = true)]
+    [PackageRegistration(UseManagedResourcesOnly = true, AllowsBackgroundLoading = true)]
     [Guid(Guids.InlineReviewsPackageId)]
     [ProvideAutoLoad(UIContextGuids80.SolutionExists)]
```

Comment:
This will cause the package to auto-load on the Main thread. I think we should be using this:
```
    [ProvideAutoLoad(UIContextGuids80.SolutionExists, PackageAutoLoadFlags.BackgroundLoad)]
```

In future we might want to only auto-load when there is an active Git repository (see #1512)?
```
    [ProvideAutoLoad(Guids.UIContext_Git, PackageAutoLoadFlags.BackgroundLoad)]
```